canary/apps/angular/angularcli/angular.json

```diff
@@ -18,15 +18,20 @@
             "main": "src/main.ts",
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
-            "aot": true,
             "assets": [
               "src/favicon.ico",
               "src/assets"
             ],
             "styles": [
               "src/styles.css"
             ],
-            "scripts": []
+            "scripts": [],
+            "vendorChunk": true,
+            "extractLicenses": false,
+            "buildOptimizer": false,
+            "sourceMap": true,
```

Comment:
Do we want to enable sourceMaps? This will increase the package size, right?

Reply:
This is only for canary app, not library. This was auto-generated, so leaving this as is to mimic customer apps